library/package.json

```diff
@@ -1,69 +1,78 @@
 {
-  "name": "rwieruch-react-library",
-  "version": "0.0.1",
-  "description": "rwieruch-react-library",
+  "name": "mh-react-library",
+  "version": "0.0.4",
+  "description": "mh-react-library",
   "main": "lib/main.js",
-  "directories": {
-    "lib": "lib"
-  },
-  "resolutions": {
-    "@babel/preset-env": "7.5.5"
-  },
+  "files": [
```

Comment:
Is this rollup specific?

Reply:
Nope, it is from package.json: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files

Reply:
Ah okay! I thought that's why we had the .npmignore file. But this seems to be another way (better way) to do it! Thanks